src/styles/main.scss

```diff
@@ -1,3 +1,28 @@
+$secondary-color: #616070;
+$main-accent-color: #060b35;
+
+@import "./blocks/header.scss";
+@import "./blocks/card.scss";
+@import "./blocks/button.scss";
+@import "./blocks/price.scss";
+@import "./blocks/starsPlusReviews.scss";
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+}
+
 body {
   margin: 0;
 }
+
+.cardContainer {
```

Comment:
Check the 8th point from the [checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md?rgh-link-date=2022-03-08T15%3A54%3A52Z)

Reply:
ok. let's create one more file